index.html

```diff
@@ -8,50 +8,266 @@
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/availability-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Multi-Post Stories</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">Facebook</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Full Stack Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">Ruby on rails</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/gym-fit-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
```

Comment:
- [ ] All the projects technologies should be wrapped with <ul> <li> tags, eg.:
``` markup
<ul>
  <li>JavaScript</li>
</ul>
```
Kindly change this.